File: src/siminov/hybrid/phonegap/template/events/DatabaseEventHandler.java
package siminov.hybrid.phonegap.template.events;

import siminov.orm.events.IDatabaseEvents;
import siminov.orm.model.DatabaseDescriptor;
import siminov.orm.model.DatabaseMappingDescriptor;
import siminov.orm.model.DatabaseMappingDescriptor.Index;

public class DatabaseEventHandler implements IDatabaseEvents {

	public void databaseCreated(DatabaseDescriptor databaseDescriptor) {
		System.out.print("");
	}

	public void databaseDropped(DatabaseDescriptor databaseDescriptor) {
		System.out.print("");
	}

	public void tableCreated(DatabaseDescriptor databaseDescriptor,
			DatabaseMappingDescriptor databaseMapping) {
		System.out.print("");
	}

	public void tableDropped(DatabaseDescriptor databaseDescriptor,
			DatabaseMappingDescriptor databaseMapping) {
		System.out.print("");
	}

	public void indexCreated(DatabaseDescriptor databaseDescriptor,
			DatabaseMappingDescriptor databaseMapping, Index index) {
		System.out.print("");
	}

	public void indexDropped(DatabaseDescriptor databaseDescriptor,
			DatabaseMappingDescriptor databaseMapping, Index index) {
		System.out.print("");		
	}

}

File: SIMINOV-HYBRID-PHONEGAP-TEMPLATE/assets/www/Liquor/Events/DatabaseEventHandler.js

function DatabaseEventHandler() {

    this.onDatabaseCreated = function(databaseDescriptor) {
//        alert("databaseCreated");
    }


    this.onDatabaseDropped = function(databaseDescriptor) {
//        alert("databaseDropped");
    }


    this.onTableCreated = function(databaseDescriptor, databaseMappingDescriptor) {
//        alert("tableCreated");
    }


    this.onTableDropped = function(databaseDescriptor, databaseMappingDescriptor) {
//        alert("tableDropped");
    }


    this.onIndexCreated = function(databaseDescriptor, databaseMappingDescriptor, index) {
//        alert("indexCreated");
    }


    this.onIndexDropped = function(databaseDescriptor, databaseMappingDescriptor, index) {
//        alert("indexDropped");
    }

}
